storybook/components/LinkTo/LinkTo.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { HTMLAttributes } from "react"
+import { linkTo } from "@storybook/addon-links"
+import classnames from "classnames"
+
+interface LinkToProps extends HTMLAttributes<HTMLButtonElement> {
+  pageId: string
+}
+
+export const LinkTo = ({
+  children,
+  pageId,
+  className,
+  ...restButtonAttributes
+}: LinkToProps): JSX.Element => (
+  <button
+    onClick={linkTo(pageId)}
+    className={classnames(
+      "bg-transparent border-none text-blue-400 hover:underline cursor-pointer p-0",
```

Comment:
Shock, Tailwind!

Reply:
Honestly almost gave up on the idea 😂 But I figured maybe I should try it.
I do hate that I couldn't set `background: none` and needed to use `bg-transparent` though because `bg-none` is `background-image: none` 🙄